compute/vm-image-spec-bookworm.yaml

```diff
@@ -145,7 +145,7 @@ merge: |
 
   COPY compute_rsyslog.conf /etc/compute_rsyslog.conf
   RUN chmod 0666 /etc/compute_rsyslog.conf
-  RUN chmod 0666 /var/log/
+  RUN mkdir /var/log/rsyslog && chmod 777 /var/log/rsyslog
```

Comment:
NIT: I don't think it matters in terms of security, but 777 always triggers me as it means ALL aka everyone can do everything. Can it be some finer grained permission?

At least it's worth of a comment why 777 is absolutely necessary

Reply:
changed owner to postgres, this is enough